src/stories/containers/Finances/SeccionPages/CardsNavigation/CardsNavigation.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import CardsNavigation from './CardsNavigation';
+import type { ComponentMeta } from '@storybook/react';
+import type { FigmaParams } from 'storybook-addon-figma-comparator/dist/ts/types';
+
+export default {
+  title: 'Pages/NewFinances/CardsNavigation',
```

Comment:
as this is a component and not a page, the right place to live would be in `components/NewFinance/...`

Reply:
done